nomad/assets/dashboards/overview.json

```diff
@@ -0,0 +1,867 @@
+            },
+            "title": true,
+            "title_align": "center",
+            "title_size": 16,
+            "title_text": "# Goroutines",
+            "type": "query_value",
+            "width": 20,
+            "x": 42,
+            "y": 58
+        },
+        {
+            "height": 10,
+            "tile_def": {
+                "autoscale": true,
+                "custom_links": [],
+                "precision": 2,
+                "requests": [
+                    {
+                        "aggregator": "avg",
+                        "q": "avg:nomad.runtime.heap_objects{$host}"
+                    }
+                ],
+                "viz": "query_value"
+            },
+            "title": true,
+            "title_align": "center",
+            "title_size": 16,
+            "title_text": "# Heap Objects",
+            "type": "query_value",
+            "width": 20,
+            "x": 42,
+            "y": 71
+        },
+        {
+            "height": 10,
+            "tile_def": {
+                "autoscale": true,
+                "custom_links": [],
+                "precision": 2,
+                "requests": [
+                    {
+                        "aggregator": "avg",
+                        "q": "avg:nomad.runtime.alloc_bytes{$host}"
+                    }
+                ],
+                "viz": "query_value"
+            },
+            "title": true,
+            "title_align": "center",
+            "title_size": 16,
+            "title_text": "Mem Utilization",
+            "type": "query_value",
+            "width": 20,
+            "x": 42,
+            "y": 84
+        },
+        {
+            "height": 10,
+            "tile_def": {
+                "autoscale": true,
+                "custom_links": [],
+                "precision": 2,
+                "requests": [
+                    {
+                        "aggregator": "avg",
+                        "q": "avg:nomad.raft.apply{$host}"
+                    }
+                ],
+                "viz": "query_value"
+            },
+            "title": true,
+            "title_align": "center",
+            "title_size": 16,
+            "title_text": "#Transactions",
+            "type": "query_value",
+            "width": 20,
+            "x": 63,
+            "y": 58
+        },
+        {
+            "height": 10,
+            "tile_def": {
+                "autoscale": true,
+                "custom_links": [],
+                "precision": 2,
+                "requests": [
+                    {
+                        "aggregator": "avg",
+                        "q": "avg:nomad.raft.appendEntries.avg{$host}"
+                    }
+                ],
+                "viz": "query_value"
+            },
+            "title": true,
+            "title_align": "center",
+            "title_size": 16,
+            "title_text": "Transaction Time",
+            "type": "query_value",
```

Comment:
Would transaction time be better as a time series? If there are multiple nomad hosts, a timeseries can show the avg transaction time for a specific period by host.